### PR TITLE
Adds a crashed pubby as a sand world ruin

### PR DESCRIPTION
**Pull request by immortalartisan**
## About The Pull Request

Adds a new pubby crash map along with the requisite area and ruin codes

## Why It's Good For The Game

It adds another ruin... this is a good thing

## Changelog

:cl: Immortalartisan
add: A new sandworld ruin based on deprecated ship pubby   
/:cl:

pubby in game light dark
SDMM view with areas
image_2023-11-10_094953863

### CODE REVIEW

**Comment by immortalartisan** [comment]
Fixed all conflicts awaiting review by a maptainer

**Review by spockye — DISMISSED** [review]
image

light on locker

image

unrotated button. mabye make things a bit more thrown about in the ship, instead of being on tables? would give it a more unkept feel

image

light on apc

image

light on locker, light on newscaster

image

light on airalarm

image

ruin has 6 apc's, yet one area! this will cause bugs. (remove all but one apc)

image

Remove the autosurgeon. while it's a cool item, its a tad too good for the challenge this ruin poses. (shouldn't really be used at all, anywhere) 


------
good work!

File: config/sandruinblacklist.txt
Comment (spockye) [inline config/sandruinblacklist.txt:1]: #_maps/RandomRuins/SandRuins/pubbysloptcrash.dmm

a T too much

**Comment by immortalartisan** [comment]
Changes made awaiting review again

File: config/sandruinblacklist.txt
Comment (Zevotech) [inline config/sandruinblacklist.txt]: not sure if the blacklist is even used (or even works for that matter) but you should probably remove the extra t between slop and crash

File: code/game/area/areas/ruins/sandplanet.dm
Comment (Zevotech) [inline code/game/area/areas/ruins/sandplanet.dm:10]: Not required but I'd personally change this to "Crashed Pubby" or "Crashed Ship" as it will show up on the crew monitor if someone is in there.

File: _maps/RandomRuins/SandRuins/whitesands_surface_pubbyslopcrash.dmm
Comment (Zevotech) [inline _maps/RandomRuins/SandRuins/whitesands_surface_pubbyslopcrash.dmm:1]: create an entry for this in the map catalogue in _maps/map_catalogue.txt
use the map catalogue key (found in the same place) for instructions and information on what to put in there.

**Comment by Zevotech** [comment]
image
Mysterious area/space. If its not there on your end feel free to ignore this.

**Comment by Zevotech** [comment]
image
Don't use micro electric engine precharger boards, use the normal ones. The micro ones are used only on the Pill and have a density of 0 (bad).

**Comment by Zevotech** [comment]
image
Floating firealarm

File: code/game/area/areas/ruins/sandplanet.dm
Comment (immortalartisan) [reply code/game/area/areas/ruins/sandplanet.dm:10]: Im going to choose to leave it as be as i think crashed ship is very too generic to use as a area name

**Review by goober3 — CHANGES_REQUESTED** [review]
image
please beef up these rocks around the corners a little. one-wide mountains will look odd -- and reveal structures behind them, as they do not blend with walls. 

image
replace these sand turfs with /lit variants -- or replace them with rock turfs, if you want to make this look like a cave.

good job! we'll get this test merged soon.

**Comment by immortalartisan** [comment]
Requested changes made this pull request is ready to be merged once the tests are run then correct?

**Comment by goober3** [comment]
essentially, yes.

**Comment by immortalartisan** [comment]
Checks are all green time to test!

**Review by goober3 — APPROVED** [review]

**Comment by goober3** [comment]
i'll try to get this ruin spawned during the uptime, today! if it goes without a hitch, we'll merge this. good job!!

**Review by goober3 — CHANGES_REQUESTED** [review]
image
image
decals & doors inside these rocks -- fix this, please!

image
remove this painting frame!

**Comment by Zevotech** [comment]
There are still podlocks and blast doors under the rocks in the pod bay area.

**Review by spockye — CHANGES_REQUESTED** [review]
what Zevo said. i'll Merge once you fix it

File: config/sandruinblacklist.txt
Comment (immortalartisan) [reply config/sandruinblacklist.txt:1]: there isnt a t for me

**Comment by Latentish** [comment]
Please add a screenshot from SDMM with the areas on.

**Comment by spockye** [comment]
fix the conflict, and i'll see about merging this!

**Comment by immortalartisan** [comment]
> Please add a screenshot from SDMM with the areas on.

Screen shot added to PR

**Review by Erikafox — CHANGES_REQUESTED** [review]
image
take a look at the rest of our crash maps and see how they handle crashes. Usually you can see that they landed, pushing up a crater, as opposed to being entombed in rock.

Once you do that, you can scatter some more debris around and create more of an "impact" in your story.

The base idea is sound, if not exciting. Not every ruin needs to be some world shattering revelation though, so I happen to like this.

**Review by goober3 — APPROVED** [review]

**Comment by goober3** [comment]
I have a few more nitpicks on this, but seeing as this has been open since september. marged.

**Pull request merged**
